hudi-aws/src/main/java/org/apache/hudi/config/GlueCatalogSyncClientConfig.java

```diff
@@ -97,4 +103,36 @@ public class GlueCatalogSyncClientConfig extends HoodieConfig {
       .markAdvanced()
       .withDocumentation("Glue sync may fail if the Glue table exists with partitions differing from the Hoodie table or if schema evolution is not supported by Glue."
           + "Enabling this configuration will drop and create the table to match the Hoodie config");
+
+  public static final ConfigProperty<String> GLUE_CATALOG_ID = ConfigProperty
+      .key(GLUE_CLIENT_PROPERTY_PREFIX + "catalogId")
+      .noDefaultValue()
+      .sinceVersion("0.15.0")
+      .markAdvanced()
+      .withDocumentation("The catalogId needs to be populated for syncing hoodie tables in a different AWS account");
+
+  public static final ConfigProperty<String> GLUE_SYNC_DATABASE_NAME = ConfigProperty
+      .key(GLUE_CLIENT_PROPERTY_PREFIX + "database_name")
+      .noDefaultValue()
+      .withInferFunction(cfg -> Option.ofNullable(cfg.getString(META_SYNC_DATABASE_NAME.key()))
```

Comment:
can we add sinceVersion for this and next config as well

Reply:
can add sinceVersion, what did you mean by next config?